tests/test_visitors/test_ast/test_builtins/test_assign/test_unpacking_rules.py

```diff
@@ -71,6 +72,11 @@ def wrapper():
         ...
 """
 
+single_destructuring1 = 'first = {0}'
```

Comment:
Let's use `correct_` and `wrong_` prefixes for examples, so we would know which is which.